src/inference/src/dev/plugin.cpp

```diff
@@ -33,6 +33,10 @@ void ov::Plugin::set_name(const std::string& deviceName) {
     });
 }
 
+const std::string ov::Plugin::get_name() const {
```

Comment:
what's this API for?

Reply:
It is used to get device name from plugin object during ov_core handling core properties for plugin.